G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/PingCommand.md

```diff
@@ -1,41 +1,33 @@
 # ping
 
-**Syntax:**
+## Syntax
 
 ```G1ANT
-ping
+ping ip ⟦text⟧ repeats ⟦integer⟧
 ```
 
-**Description:**
+## Description
 
-Command `ping` allows to ping specified IP and receives approximate round-trip time in milliseconds.
+This command pings a specified IP address and returns an approximate round-trip time in milliseconds.
 | Argument | Type | Required | Default Value | Description |
 | -------- | ---- | -------- | ------------- | ----------- |
-|`ip`| [string](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Structures/string.md)  | no  | 8.8.8.8 | IP address or hostname of pinging server |
-|`repeats`| [integer](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Structures/integer.md)  | no |  | allows to ping multiple times, command returns rounded value of all pings. Default = 1 |
-|`result`| [variable](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Special-Characters/variable.md)  | no | [♥result](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Common-Arguments.md)  | name of variable where approximate round trip time in milliseconds will be stored |
-|`if`| [bool](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Structures/bool.md) | no | true | runs the command only if condition is true |
-|`timeout`| [variable](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Special-Characters/variable.md) | no | [♥timeoutcommand](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Variables/Special-Variables.md)  | specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
-|`errorjump` | [label](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Structures/label.md) | no | | name of the label to jump to if given `timeout` expires |
-|`errormessage`| [string](https://github.com/G1ANT-Robot/G1ANT.Manual/blob/master/G1ANT-Language/Structures/string.md) | no |  | message that will be shown in case error occurs and no `errorjump` argument is specified |
+|`ip`| [text](G1ANT.Language/G1ANT.Language/Structures/TextStructure.md) | no  | 8.8.8.8 | IP address or a host name of a pinged server |
```

Comment:
The table is not beeing created by github .md preview.

Reply:
corrected